content/blog/2018/06/2018-05-14-simple-pull-request-plugin.adoc

```diff
@@ -0,0 +1,241 @@
+---
+layout: post
+title: "GSoC Project Intro: Simple pull request Plugin"
+tags:
+- gsoc2018
+- plugin
+author: abhishek_gautam
+---
+
+=== About me
+
+I am Abhishek Gautam, 3rd year student from Visvesvaraya National Institute of
+technology, India, Nagpur. I was a member of ACM Chapter and Google student developer club of his
+college. I am passianate about automation.
+
+=== Mentors
+. https://github.com/oleg-nenashev[Oleg Nenashev] (Org Admin)
+
+. https://github.com/martinda[Martin d'Anjou]
+. https://github.com/kwhetstone[Kristin Whetstone]
+. https://github.com/Jeff-Symphony[Jeff Knurek]
+
+
+=== Project Summary
+This is a GSoC 2018 project.
+
+This project aims to develop a Pull request Job Plugin. Users should be able to
+configure job type using YAML file which will be placed in root directory of the
+Git repository being the subject of the PR. Plugin should interact with various
+platfoms like Bitbucket, Github, Gitlab, etc whenever a pull request is created or updated.
+
+Detect the presence of certain types of the report based on a conventional location,
+and automatically publish them. If the reports are not in a conventional location,
+users could specify the location using the YAML file.
+
+==== Benefits to the community
+* Project administrators will be able to handle builds for pull requests more easily.
+* Build specifications for pull request can be written in a more readable format.
+* Build reports will be automatically published to Github, Bitbucket, ect.
+* Build status updates will be sent to git servers automatically.
+* Users will not have to deal with pipeline code.
+
+=== Prior work
+. Yaml Axis Plugin: This plugin is designed for creation and exclusion of
+  properties in Matrix project. It uses YAML for that.
+
+. Travis YML Plugin: Designed to run .travis.yml as jenkins pipeline job.
+  Travis-CI does not support external pull requests hence. And jenkins environment
+  is different than Travis and does not always make sense to use configurations
+  defined for other environment in jenkins. Also maintenance of this is slowed
+  down and last commit for this plugin was on 14 Nov 2016.
+  https://github.com/jenkinsci/travis-yml-plugin/commits/master[Click here] to check.
+
+. CodeShip Plugin: This plugin is designed to convert codeship "steps.yaml" and
+  "services.yaml" to scripted pipeline code.
+
+
```

Comment:
i would add https://github.com/constantcontact/jenkins_pipeline_builder to the list